scripts/test.js

```diff
@@ -4,6 +4,27 @@ require("@app/config/env");
 const { execSync, spawnSync: rawSpawnSync } = require("child_process");
 const concurrently = require("concurrently");
 
+// Dear graphile-migrate, please treat the test DB as if it were the shadow DB
+process.env.SHADOW_DATABASE_URL = process.env.TEST_DATABASE_URL;
+
+// Signal to graphile-migrate scripts that we're in the tests
+process.env.IN_TESTS = "1";
+process.env.NODE_ENV = "test";
+
+const cmdArgs = process.argv;
+const watchMode = cmdArgs.find(
+  (arg) => arg === "--watch" || arg === "--watchAll"
+);
+const delayArg = cmdArgs.indexOf("--delay");
+let delaySeconds = null;
+if (delayArg > 0) {
+  delaySeconds = parseInt(cmdArgs[delayArg + 1]);
```

Comment:
When using `parseInt` **always** pass the radix:

```suggestion
  delaySeconds = parseInt(cmdArgs[delayArg + 1], 10);
```

Reply:
Actually, let's use parseFloat here so you could do e.g. `0.25` seconds

```suggestion
  delaySeconds = parseFloat(cmdArgs[delayArg + 1]);
```